src/lib/ppx_coda/tests/dune

```diff
@@ -63,6 +62,6 @@
   (modules versioned_bad_type_name))
 
 (library
-  (name versioned_bad_with_option)
+  (name versioned_bad_option)
```

Comment:
~~Tests are failing because of this, I think you need to add the module file.~~

Edit: Too fast for me, you already fixed it :smile: